jsapp/scss/components/_kobo.map.scss

```diff
@@ -230,95 +275,104 @@ body .leaflet-control-layers,
 // marker colors
 
 .map-marker {
-  width: 30px;
-  height: 30px;
+  width: 24px;
+  height: 24px;
   background: rgba(255, 50, 0, 0.4);
   border: 1px solid #FFF;
-  border-radius: 30px;
-}
+  border-radius: 50%;
+  position: relative;
 
-.map-marker {
-  &-0, &-0:after {
-    background: rgba(255, 50, 0, 0.4);
+  &.unselected {
+    visibility: hidden;
   }
+}
 
-  &-1, &-1:after {
-    background: rgba(78, 203, 255, 0.4);
-  }
+.map-marker[class*="map-marker-"] {
+  background: rgba(0, 0, 0, 0.25);
+  border-color: rgba(255, 255, 255, 0.75);
+}
 
-  &-2, &-2:after {
-    background: rgba(255, 180, 50, 0.4);
-  }
+// OLD COLOR LIST
+// $color-collection: rgba(255, 50, 0, 0.4), rgba(78, 203, 255, 0.4), rgba(255, 180, 50, 0.4), rgba(113, 230, 33, 0.4), rgba(253, 35, 4, 0.4), rgba(33, 231, 184, 0.4), rgba(55, 40, 210, 0.4), rgba(200, 80, 246, 0.4), rgba(208, 165, 50, 0.4), rgba(232, 65, 14, 0.4), rgba(216, 150, 216, 0.4), rgba(200, 233, 150, 0.4), rgba(200, 233, 150, 0.4), rgba(122, 187, 200, 0.4), rgba(90, 90, 90, 0.4), rgba(60, 223, 219, 0.4), rgba(52, 106, 200, 0.4), rgba(88, 20, 100, 0.4), rgba(50, 90, 10, 0.4), rgba(10, 180, 80, 0.4), rgba(230, 175, 25, 0.4);
 
-  &-3, &-3:after {
-    background: rgba(252, 74, 124, 0.4);
-  }
+$color-collection: rgba(255, 50, 0, 0.4), rgba(125, 245, 46, 0.4), rgba(86, 0, 178, 0.4), rgba(0, 236, 118, 0.4), rgba(202, 0, 188, 0.4), rgba(104, 155, 0, 0.4), rgba(161, 107, 255, 0.4), rgba(191, 184, 0, 0.4), rgba(0, 66, 187, 0.4), rgba(197, 231, 113, 0.4), rgba(239, 0, 161, 0.4), rgba(2, 231, 203, 0.4), rgba(239, 0, 69, 0.4), rgba(2, 153, 148, 0.4), rgba(43, 130, 255, 0.4), rgba(183, 142, 0, 0.4), rgba(60, 46, 126, 0.4), rgba(35, 106, 0, 0.4), rgba(175, 158, 255, 0.4), rgba(198, 116, 0, 0.4), rgba(0, 140, 218, 0.4), rgba(255, 109, 60, 0.4), rgba(1, 76, 140, 0.4), rgba(184, 69, 0, 0.4), rgba(165, 228, 255, 0.4), rgba(112, 31, 9, 0.4), rgba(163, 231, 229, 0.4), rgba(133, 0, 75, 0.4), rgba(167, 234, 188, 0.4), rgba(96, 33, 97, 0.4), rgba(21, 70, 0, 0.4), rgba(255, 110, 148, 0.4), rgba(0, 87, 84, 0.4), rgba(255, 157, 210, 0.4), rgba(51, 64, 37, 0.4), rgba(255, 153, 153, 0.4), rgba(12, 64, 93, 0.4), rgba(255, 194, 186, 0.4), rgba(96, 42, 58, 0.4), rgba(134, 0, 40, 0.4);
```

Comment:
Are those some generated colors - if yes, maybe add source url?

Reply:
I added a link to the generator. This will soon be replaced with a configurable list of colors though, similar to reports.